Package.swift

```diff
@@ -0,0 +1,74 @@
+// swift-tools-version:5.4.0
+
+import PackageDescription
+
+let package = Package(
+    name: "ably-cocoa",
+    platforms: [
+        .macOS(.v10_11),
+        .iOS(.v9),
+        .tvOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "ably-cocoa",
+            targets: ["ably-cocoa"]
```

Comment:
You shouldn't make the target name `ably-cocoa`, because now users have to `import ably-cocoa`. This should remain the. same as before, so imports are unchanged, `import Ably`. The package name is different to the target name, which is different the library name.

Reply:
changed: https://github.com/ably/ably-cocoa/pull/1136/commits/18b0b100e953cdcc446272aa403d2a31ceb91cfb